xbmc/pvr/recordings/PVRRecording.cpp

```diff
@@ -376,6 +378,7 @@ void CPVRRecording::Update(const CPVRRecording &tag)
   m_bIsDeleted        = tag.m_bIsDeleted;
   m_iEpgEventId       = tag.m_iEpgEventId;
   m_iChannelUid       = tag.m_iChannelUid;
+  m_iRecordingState   = tag.m_iRecordingState;
```

Comment:
assert() if value is PVR_RECORDED_TV_ITEM_STATE_IN_PROGRESS! We must force the addon devs to support this property.